src/ActiveLogin.Authentication.BankId.AspNetCore.QRCoder/BankIdMissingQrCodeGenerator.cs

```diff
@@ -1,16 +1,17 @@
 using System.Collections.Generic;
+using ActiveLogin.Authentication.BankId.AspNetCore.Qr;
 using Microsoft.AspNetCore.Http.Extensions;
 using QRCoder;
 
-namespace ActiveLogin.Authentication.BankId.AspNetCore.Models
+namespace ActiveLogin.Authentication.BankId.AspNetCore.QrCoder
 {
     /// <summary>
     /// This class generates QR codes for BankID
     /// </summary>
     /// <remarks>
     /// It uses the QR Coder library to generate the codes
     /// </remarks>
-    public class QRCoderBankIdQrCodeGenerator : IBankIdQrCodeGenerator
+    public class BankIdMissingQrCodeGenerator : IBankIdQrCodeGenerator
```

Comment:
I think you renamed the wrong file. this should be called:
`QrCoderBankIdQrCodeGenerator`

And the "default" one should be called:
`BankIdMissingQrCodeGenerator `

Reply:
lol